airbyte-integrations/connectors/source-fastbill/manifest.yaml

```diff
@@ -0,0 +1,2770 @@
+            - string
+            empty: true
+  products_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+          - "RESPONSE"
+          - "{{ parameters.record_extractor }}"
+      paginator:
+        type: "DefaultPaginator"
+        pagination_strategy:
+          type: "OffsetIncrement"
+          page_size: 100
+        page_token_option:
+          type: "RequestOption"
+          field_name: "OFFSET"
+          inject_into: "body_json"
+      requester:
+        type: HttpRequester
+        url_base: "https://my.fastbill.com/api/1.0"
+        http_method: "POST"
+        authenticator:
+          class_name: source_declarative_manifest.components.CustomAuthenticator
+          username: "{{config['username']}}"
+          password: "{{config['api_key']}}"
+        request_body_json:
+          SERVICE: "{{ parameters.endpoint }}.get"
+          LIMIT: 100
+          Content-Type: "application/json"
+    name: "products"
+    primary_key: "ARTICLE_ID"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $schema: http://json-schema.org/draft-04/schema#
+        type: object
+        properties:
+          ARTICLE_ID:
+            description: Unique identifier for the product article.
+            type: string
+          ARTICLE_NUMBER:
+            description: Identification number for the product article.
+            type: string
+          TITLE:
+            description: Title or name of the product article.
+            type: string
+          DESCRIPTION:
+            description: Detailed description of the product article.
+            type: string
+            empty: true
+          UNIT:
+            description: Unit of measurement for the product article (e.g., piece,
+              kg).
+            type: string
+          UNIT_PRICE:
+            description: Price per unit of the product article.
+            type: string
+          CURRENCY_CODE:
+            description: The currency code used for the price of the product article.
+            type: string
+          VAT_PERCENT:
+            description: The percentage of Value Added Tax applied to the product
+              article price.
+            type: string
+          IS_GROSS:
+            description: Indicates whether the price is gross or net (inclusive of
+              tax).
+            type: number
+          TAGS:
+            description: Tags associated with the product article.
+            type:
+            - string
+            - "null"
+            empty: true
+  revenues_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+          - "RESPONSE"
+          - "{{ parameters.record_extractor }}"
```

Comment:
This is indicative of a bug in the migration script 😕 The use of parameters isn't compatible with the builder UI, so we have to resolve them when migrating to manifest-only. This should happen automatically but I'm seeing more examples where it failed. I'm looking into the issue, but for now we can try this flow to resolve:

1. Grab this manifest, throw it in the builder and switch to UI mode.
2. Look for any fields that reference parameters. Check the existing manifest in Github and update the values for each stream ([example here](This is a bug in the migration script 😕 The use of parameters isn't compatible with the builder UI, so we have to resolve them when migrating to manifest-only. This should happen automatically but I'm seeing more examples where it failed. I'm looking into the issue, but for now we can try this flow to resolve:

1. Grab this manifest, throw it in the builder and switch to UI mode.
2. Look for any fields that reference parameters. Check the existing manifest in Github and update the values for each stream [(example here)](https://github.com/airbytehq/airbyte/blob/1c2f72ec09200b0a3bc04f2fd37b990e9dadd82d/airbyte-integrations/connectors/source-fastbill/source_fastbill/manifest.yaml#L48)
3. Switch back to YAML view and copy the updated manifest for the PR.

Hopefully that will resolve it. Let me know if anything about this is unclear!

Reply:
Updates done!, Hopefully it works

Reply:
Looks like that fixed it, nice work @btkcodedev!